contracts/core/Core.sol

```diff
@@ -835,9 +831,9 @@ contract Core is
 
         nextKernelHeight_ = openKernelHeight.add(1);
 
-        // removeValidator performs necessary requirements
+        // removeValidatorInternal asserts validator is currently in the validator set and not already logged out.
```

Comment:
This comment is exceeding the line break limit of 79.
For multiple line comments we use 
```
/*
 * Remove validator from next metablock height.
 * removeValidatorInternal asserts validator is currently in the
 * validator set and not already logged out.
 */
```

Reply:
Done.